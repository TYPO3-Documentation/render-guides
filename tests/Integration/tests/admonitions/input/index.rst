
===============================================
Admonitions: Tip, Note, Warning, See also,  etc
===============================================

..  admonition:: Custom admonition

    Some Custom admonitions

See also
--------

..  seealso::
    `Admonitions <http://docutils.sourceforge.net/0.7/docs/ref/rst/directives.html#admonitions>`__


Note
----

..  note::
    A note


..  index:: reST directives; tip

Tip
---

..  tip::
    A tip

..  hint::
    A Hint

..  index:: reST directives; warning

Warning
-------

..  warning::
    Some text pointing out something that people should be warned about.

..  caution::
    Some text pointing out something that people should be cautioned about.

..  danger::
    Some dangerous text pointing out something that people should be warned about.

..  index:: reST directives; attention

Attention
---------

..  attention::
    A attention
