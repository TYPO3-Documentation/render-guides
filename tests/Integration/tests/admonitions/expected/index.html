<!-- content start -->
                <section class="section" id="admonitions-tip-note-warning-see-also-etc">
            <h1>Admonitions: Tip, Note, Warning, See also,  etc<a class="headerlink" href="#admonitions-tip-note-warning-see-also-etc" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">¶</a></h1>


<div class="admonition custom-admonition" role="alert">
        <p class="admonition-title">Custom admonition</p>
            <p>Some Custom admonitions</p>
</div>


            <section class="section" id="see-also">
            <h2>See also<a class="headerlink" href="#see-also" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">¶</a></h2>


<div class="admonition seealso" role="alert">
    <p class="admonition-title">See also</p>
                <p><a href="http://docutils.sourceforge.net/0.7/docs/ref/rst/directives.html#admonitions">Admonitions</a></p>
</div>


    </section>

            <section class="section" id="note">
            <h2>Note<a class="headerlink" href="#note" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">¶</a></h2>


<div class="admonition note" role="alert">
    <p class="admonition-title">Note</p>
                <p>A note</p>
</div>


    </section>

            <section class="section" id="tip">
            <h2>Tip<a class="headerlink" href="#tip" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">¶</a></h2>


<div class="admonition tip" role="alert">
    <p class="admonition-title">Tip</p>
                <p>A tip</p>
</div>



<div class="admonition hint" role="alert">
    <p class="admonition-title">Hint</p>
                <p>A Hint</p>
</div>



<div class="admonition hint" role="alert">
    <p class="admonition-title">Hint</p>
            <p>One line hint</p>

</div>


    </section>

            <section class="section" id="warning">
            <h2>Warning<a class="headerlink" href="#warning" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">¶</a></h2>


<div class="admonition warning" role="alert">
    <p class="admonition-title">Warning</p>
                <p>Some text pointing out something that people should be warned about.</p>
</div>



<div class="admonition caution" role="alert">
    <p class="admonition-title">Caution</p>
                <p>Some text pointing out something that people should be cautioned about.</p>
</div>



<div class="admonition danger" role="alert">
                <p>Some dangerous text pointing out something that people should be warned about.</p>
</div>


    </section>

            <section class="section" id="attention">
            <h2>Attention<a class="headerlink" href="#attention" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">¶</a></h2>


<div class="admonition attention" role="alert">
    <p class="admonition-title">Attention</p>
                <p>A attention</p>
</div>


    </section>

    </section>

        <!-- content end -->
